### PR TITLE
Feature/set password

### PR DESCRIPTION
**Pull request by akaFTS**
#### What is the purpose of this pull request?
This pull request implements a feature to redefine passwords, and also fixes some error handling cases and upgrades all color-related CSS classes to tokens.

#### Screenshots or example usage
captura de tela 2018-08-29 as 17 41 14

#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [X] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/locales/pt-BR.json [hunk 1]
```diff
@@ -49,10 +55,15 @@
...
   "commons.edit": "Editar",
 
+  "alert.unauthenticated": "Você precisa se identificar para realizar esta ação.",
   "alert.connectionError": "Houve um erro de comunicação com o sistema, tente recarregar a página.",
   "alert.addressNotFound": "O endereço especificado não foi encontrado.",
   "alert.unknownError": "Ocorreu um erro, tente novamente.",
   "alert.success": "Suas informações foram salvas com sucesso.",
   "alert.reload": "Recarregar",
-  "alert.fileTooBig": "Esta imagem ultrapassa o limite de 4MB, selecione um menor."
+  "alert.fileTooBig": "Esta imagem ultrapassa o limite de 4MB, selecione um menor.",
```
Comment (Caio-Batista) [inline react/locales/pt-BR.json]: Typo here, _'uma'_ not _'um'_.

File: react/graphql/redefinePassword.gql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+mutation RedefinePassword($email: String!, $currentPassword: String!, $newPassword: String!) {
```
Comment (Caio-Batista) [inline react/graphql/redefinePassword.gql]: Isn't this code repetition?  In a previous commit the function SetPassword has the same structure.

Comment (akaFTS) [reply react/graphql/redefinePassword.gql]: That's the same file, I just altered the mutation name

File: react/locales/pt-BR.json (hunk 1 above)
Comment (akaFTS) [reply react/locales/pt-BR.json]: Good catch!

File: react/graphql/redefinePassword.gql (hunk 2 above)
Comment (Caio-Batista) [reply react/graphql/redefinePassword.gql]: Nice, thanks.

File: manifest.json [hunk 3]
```diff
@@ -20,12 +18,6 @@
     "pages": "0.x",
     "node": "3.x"
   },
-  "billingOptions": {
```
[the remark this answers is not in the document]
Comment (brunojdo) [reply manifest.json]: Pq app padrão nao deveria ter billing @klzns Essa app é tipo my-orders, carousel, shelf, login. Acho que gustavo colocou o billing pra poder ver as configurações la no `admin/apps` só que atualmente ja foi modificado pra exibir as configurações se tiver o `settingsSchema`. Antes checava se tinha billing.

**Pull request merged**